### ISSUE TITLE
Unable to Install apps trough Win Util after installing Microwin ISO

### ISSUE THREAD

**Issue by Juanc179** [ChrisTitusTech/winutil#1289]
Hi everyone, first i would like to thank you for all the work given to make this awesome utility. i've used it for the last year.
**Description**
I succesfully installed my microwin iso on a clean drive, i initialized windows as usual, the changes on the iso were succesfull. then i ran the utility as usual and the programs didn't install, the progress bar never started. i left it for 1 hour and it didnt do anything. I restarted my pc and tried again, left it for one hour and same result. the progress bar was empty. the complex thing about this issue is that there were to many things i could JUST NOT DO. 

**To Reproduce**
1. downloaded from ms page win 11 iso. 64bit International English
2. created iso with microwin only leaving Defender and Selected Windows Pro(i think this might be the problem)
3. copied the iso into a Ventoy Drive
4. Installed Windows succesfully, accidentally ran windows update : C(intalled dev home and Outlook, also might be a problem)
5.*Run the Win Util (manually coping the link from my phone, didnt notice the acces from desktop...)
6. Tried to install: Chrome, Discord, Acrobat, Inkscape, Winrar, Obs....etc.
7. installation never started, the progress bar froze at 0% installing Inkscape 
8. restart and returned to 5*

**Comment by bhatiak2050** [comment]
I've noticed this issue on a brand new windows 11 stock install as well: On some newer versions of windows, the winget utility is broken. There's an option within this tool itself to reinstall winget. I did that and then I could install programs normally using winutil.

**Comment by casual-coderr** [comment]
Yes, Just go to config and click on WinGet Reinstall and it solved the issue

**Comment by Juanc179** [comment]
Thank you for your response. I had another problem during the process of reinstalling winget. Apparently it froze, the utility doesn't respond and i cant do anything. ![20231217_195651.jpg](https://github.com/ChrisTitusTech/winutil/assets/85572320/ce8b9a5f-0087-4653-b47c-ee9fb4651c6d)

Any clues?

**Comment by casual-coderr** [comment]
Try to open another Terminal with admin and use "winget update" and post what happens. If you see list of packages to update then you can try to use the install tool once again.

**Comment by Juanc179** [comment]
Hi, here is an update.

(from the Last image) I left the pc running the utility for an hour and it eventually continued installing things, it was not frozen apparently. The Last message was "packages 4/4 installed" or something similar.

If i try to install programs it doesn't work, it happens the same as before. Then if i try reinstalling winget again, it tells me that winget is already installed, and shows me commands to force it but i cant write on the terminal.

**Comment by casual-coderr** [comment]
Yes, Now I'm also facing issues with the OS created using microwin, So i ended up using NTLite to manually configure stuff from the win 11 iso and used that for the fresh install. After installing a debloated windows, I once ran the chris titus tool's tweaks. You can checkout NTLite. I found this video helpful https://www.youtube.com/watch?v=_gMJNQ3yWNE

**Comment by Juanc179** [comment]
Thank you for your response. 

I found something interesting. The microwin install works fine, the utility is broken on any New sistem.

I installed a New windows iso, "the original one" without passing trough microwin and with winget reinstall it doesn't work neither. The only thing that recovers the utility is the first reset after installing windows. It updates something and makes the utility work as usual. Then i reinstalled winget and the install features work again.

The difference between this 2 stages of not working is that in a microwin install when it doesn't work, it just stops doing anything. The utility doesn't show any work on terminal but apparently it is doing something in background. 

Then in a normal install it just doesn't download anything and when it "finishes" it gives you the pop up, but programs don't get installed. Then after winget reinstall and reseting the pc, it works as usual.
